### DISCUSSION TITLE
Release of 0.13.0

### DISCUSSION THREAD

**Discussion by arlyon** [arlyon/async-stripe#78] [Announcements]
The library is in a good place. Here are some things we'd like to get done before 0.13.0:

- [x] port the tokio and hyper code to support 1.0 (#1)
- [x] review the API and make sure we aren't missing things from the old API
- [x] automatically update the openapi spec
- [x] remove all unwraps from the code (#20)

**Comment by kennetpostigo** [comment]
One more thing is probably updating the docs, i'm currently referencing the `stripe-rs` one.

**Comment by arlyon** [comment]
Docs should be up as of last week, still need to iron out the client documentation but the API is covered at least! :)

**Comment by kennetpostigo** [comment]
@arlyon thank you sooo much! I've been using the async-stripe so far and it has gone great!

**Comment by arlyon** [comment]
For an update here, I would like to flesh out the SetupIntent api a little then I will think about (finally) releasing 0.13.0. I just dropped 0.13.0-rc3 which reworks the error code a bit and should be the last breaking change.
